src/helpers/serverless.helper.ts

```diff
@@ -11,8 +11,7 @@ export class ServerlessHelper {
   }
 
   getPluginConfig(): GluePluginConfigInterface {
-    return this.serverless.configSchemaHandler.serverless.configurationInput
-      .Glue;
+    return this.serverless.configSchemaHandler.serverless.configurationInput.custom.Glue;
```

Comment:
This fixes a bug too

Reply:
What error are you referring to? In refactoring, plugin settings move from custom to the root

Reply:
ok

Reply:
please update the documetation.. you updated the example, but not the description.. it still says : ```Configure yours glue jobs in custom section like this:

```

Reply:
Ups, thanks I fix it